google/api/control/caches.py

```diff
@@ -250,9 +251,9 @@ def __setitem__(self, key, value, **kw):
     @property
     def out_deque(self):
         """The :class:`collections.deque` to which expired items are added."""
-        expired = dict((k, v) for (k, v) in iter(self._tracking.items())
+        expired = dict((k, v) for (k, v) in iter(list(self._tracking.items()))
```

Comment:
This can be changed to just self._tracking.items(), since that will return an iter anyway in py3, and returning a list in py2 is fine.